config/config.go

```diff
@@ -520,10 +521,15 @@ func convertToTerragruntConfig(terragruntConfigFromFile *terragruntConfigFile, c
 		}
 
 		remoteState := &remote.RemoteState{}
+		remoteState.FillDefaults()
```

Comment:
It appeared to me that `FillDefaults` should probably be called before setting any attributes? It was unused so was not fully sure of it's intention, I moved it up to execute earlier so that I could default Enabled to true when not present in config.